### DISCUSSION TITLE
documentation for Pgp module needs to be updated

### DISCUSSION THREAD

**Discussion by zhisme** [rocketjob/iostreams#19] [General]
### Environment

Provide at least:
* Gem version 1.10.3

### Expected Behavior

![image](https://user-images.githubusercontent.com/13069050/165536090-03b4db6a-1511-4a16-b0f7-f05f27f51fa8.png)
List keys function documentation outdated
`private` and `trust` should be added to the example list

### Actual Behavior
```ruby
[34] pry(main)> IOStreams::Pgp.list_keys.first.keys
=> [:private, :key_length, :key_type, :date, :key_id, :email, :name, :trust]
```

### Pull Request

* Consider submitting a Pull Request with a fix for the issue.
* Or, even a Pull request that only includes a test that reproduces the problem.

**Comment by zhisme** [comment]
also for further improvements I think `key_info` part should be updated at least with example, what expected input arguments, I mentioned them in https://github.com/reidmorrison/iostreams/issues/9#issuecomment-1111036550

and for more clear solution, I propose to keep the same output structure for data in both `key_info` and `list_keys`
so those should be equal
```ruby
[35] pry(main)> IOStreams::Pgp.list_keys.first.keys
=> [:private, :key_length, :key_type, :date, :key_id, :email, :name, :trust]
[38] pry(main)> IOStreams::Pgp.key_info(key: key).first.keys
=> [:private, :key_length, :key_type, :date, :key_id, :email, :name]
```
and why `key_info` for `key` returns an array? I expect it to be a simple hash.
let's discuss and I can propose PR for that

**Comment by reidmorrison** [comment]
I think the reason it returns an array is that for a "single" supplied pgp ascii key it can actually return  multiple keys. This is often used to publish multiple keys in a single file, often with different expiry dates.

**Reply by reidmorrison** [reply]
As for `list_keys` returning `trust`, which `key_info` does not, that is what GnuPG returns for those calls. The same code is called to parse the output from GnuPG from both calls, and trust is only returned when it is returned by GnuPG.
